src/client/pmix_client.c

```diff
@@ -964,7 +964,7 @@ static void _commitfn(int sd, short args, void *cbdata)
      * that we contributed whatever we had */
     PMIX_PTL_SEND_RECV(rc, &pmix_client_globals.myserver, msgout,
                        wait_cbfunc, (void*)&cb->active);
-    if (PMIX_SUCCESS != rc) {
+    if (PMIX_SUCCESS == rc) {
```

Comment:
This doesn't look right to me - what we should do is just fall thru so we set the status and change the flag. There is no reason to have a separate check here.